src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -1,5 +1,95 @@
-describe.skip('<Browser />', () => {
-  test('it works', () => {
-    // this is a reminder to test the Browser component
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import { Browser, BrowserProps, ExampleObjectLinks } from './Browser';
+import BrowserImage from './browser-image/BrowserImage';
+import TrackPanel from './track-panel/TrackPanel';
+import AppBar from 'src/shared/components/app-bar/AppBar';
```

Comment:
Perhaps mock all these components during the import (so that we do not render the real ones).